lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -12,14 +12,11 @@
 
```

Comment:
Interesting...  unclear where this "sysadmin" dashboard even gets used now.

Reply:
🤷‍♂️